src/chains/ethereum/options/tests/logging-options.test.ts

```diff
@@ -0,0 +1,203 @@
+import assert from "assert";
+import { EthereumOptionsConfig } from "../src";
+import sinon from "sinon";
+import { resolve } from "path";
+import { promises } from "fs";
+const { unlink, readFile, open } = promises;
+import { closeSync } from "fs";
+import { URL } from "url";
+
+describe("EthereumOptionsConfig", () => {
+  describe("logging", () => {
+    const validFilePath = resolve("./tests/test-file.log");
+
+    describe("options", () => {
+      let spy: any;
+      beforeEach(() => {
+        spy = sinon.spy(console, "log");
+      });
+
+      afterEach(() => {
+        spy.restore();
+      });
+
+      describe("logger", () => {
+        it("uses console.log by default", () => {
+          const message = "message";
+          const options = EthereumOptionsConfig.normalize({});
+          options.logging.logger.log(message);
+          assert.strictEqual(spy.withArgs(message).callCount, 1);
+        });
+
+        it("disables the logger when the quiet flag is used", () => {
+          const message = "message";
+          const options = EthereumOptionsConfig.normalize({
+            logging: { quiet: true }
+          });
+          options.logging.logger.log(message);
+          assert.strictEqual(spy.withArgs(message).callCount, 0);
+        });
+
+        it("calls the provided logger when quiet flag is used", () => {
```

Comment:
I like it as is for the reason you point out - also it becomes more complicated if the `--quiet` flag stops output to the file, because we'd maybe want to make those conflict or something, because `--quiet --logging.file` together wouldn't make sense if nothing was logged 🤷 

I've changed the description of `--quiet` to be more explicitly _only_ the console.